programs/programs/wa/orca_lift/calculator.py

```
@@ -0,0 +1,53 @@
+from typing import ClassVar
+
+import programs.programs.messages as messages
+from programs.programs.calc import Eligibility, MemberEligibility, ProgramCalculator
+from screener.models import HouseholdMember
+
+
+class WaOrcaLift(ProgramCalculator):
+    """ORCA LIFT Reduced Fare Program — discounted $1.00 transit fares for lower-income
+    adults aged 19–64 in the Puget Sound region (King, Pierce, Snohomish, Kitsap counties).
```

Ruff (0.15.13)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)